### DISCUSSION TITLE
API v2 Direct Messages

### DISCUSSION THREAD

**Discussion by brycefrank** [plhery/node-twitter-api-v2#410] [Ideas]
Twitter apparently has added support for direct messages for API v2

https://developer.twitter.com/en/docs/twitter-api/direct-messages/lookup/introduction
https://developer.twitter.com/en/docs/twitter-api/direct-messages/manage/introduction

Is there any interest in adding support for these new features?

**Comment by plhery** [comment]
Yes thank you, I saw that, I plan to add it probably next weekend (or anyone is welcome to propose a PR)
